ocs_ci/framework/__init__.py

```diff
@@ -146,6 +146,14 @@ def __init__(self):
         self.single_cluster_default = True
         self._single_cluster_init_cluster_configs()
 
+    @property
+    def default_cluster_ctx(self):
+        # Get the default index. If not found, the default value is 0
```

Comment:
Shouldn't this function also change `cur_index`? It looks like it only returns cluster config but name and comment suggest that it is worked with cluster index.

Reply:
Yes, the comment is just for the following line. As you mentioned, I will add a docstring to be more precise.